newIDE/app/src/GameDashboard/LeaderboardAdmin/LeaderboardSortOptionsDialog.js

```diff
@@ -0,0 +1,220 @@
+  };
+
+  return (
+    <I18n>
+      {({ i18n }) => (
+        <Dialog
+          open={open}
+          maxWidth="sm"
+          onRequestClose={() => {
+            if (!isLoading) onClose();
+          }}
+          secondaryActions={[
+            <HelpButton
+              key="help"
+              helpPagePath="/interface/games-dashboard/leaderboard-administration"
+              anchor="change_the_appearance_of_a_leaderboard"
+            />,
+          ]}
+          actions={[
+            <FlatButton
+              label={<Trans>Cancel</Trans>}
+              disabled={isLoading}
+              onClick={onClose}
+              key={'cancel'}
+            />,
+            <RaisedButton
+              primary
+              label={<Trans>Save</Trans>}
+              disabled={isLoading}
+              onClick={() => onSaveSettings(i18n)}
+              key={'save'}
+            />,
+          ]}
+          onApply={() => {
+            onSaveSettings(i18n);
+          }}
+        >
+          <Text size="title">
+            <Trans>Sort order</Trans>
+          </Text>
+          <ColumnStackLayout>
+            <SelectField
+              fullWidth
+              value={sortOrder}
+              floatingLabelText={<Trans>Scores sort order</Trans>}
+              onChange={(e, i, newValue) => {
+                // $FlowIgnore
+                setSortOrder(newValue);
+              }}
+            >
+              <SelectOption
+                key={'ASC'}
+                value={'ASC'}
+                primaryText={t`Lower is better`}
+              />
+              <SelectOption
+                key={'DESC'}
+                value={'DESC'}
+                primaryText={t`Higher is better`}
+              />
+            </SelectField>
+            <Checkbox
+              label={
+                <>
+                  <Line noMargin>
+                    <Trans>Limit scores</Trans>
+                  </Line>
+                  <FormHelperText style={{ display: 'inline' }}>
+                    <MarkdownText
+                      source={i18n._(
+                        `Any submitted score that is ${
+                          sortOrder === 'ASC' ? 'lower' : 'higher'
+                        } than the set value will not be saved in the leaderboard.`
+                      )}
+                    />
+                  </FormHelperText>
+                </>
+              }
+              checked={displayExtremeAllowedScoreInput}
+              onCheck={() =>
+                setDisplayExtremeAllowedScoreInput(
+                  !displayExtremeAllowedScoreInput
+                )
+              }
+            />
+          </ColumnStackLayout>
+          {displayExtremeAllowedScoreInput && (
+            <Column noMargin>
+              <Line>
+                <LargeSpacer />
+                <TextField
+                  fullWidth
+                  type="number"
+                  floatingLabelText={
+                    <Trans>
```

Comment:
In this case, you have nothing to translate. You should use:
```js
{sortOrder === 'ASC' ? <Trans>Minimum score</Trans> : <Trans>Maximum score</Trans>}
```

Reply:
Otherwise, translators will receive the string `{0}` to translate I think

Reply:
Good to know!